apps/condo/domains/resident/schema/SuggestServiceProviderService.test.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Generated by `createservice organization.SuggestServiceProviderService --type queries`
+ */
+
+const {
+    expectToThrowAuthenticationErrorToResult,
+    expectToThrowAccessDeniedErrorToResult,
+} = require('@open-condo/keystone/test.utils')
+
+
+const { CONTEXT_FINISHED_STATUS, CONTEXT_IN_PROGRESS_STATUS } = require('@condo/domains/acquiring/constants/context')
+const {
+    TestUtils,
+    ResidentTestMixin,
+    MeterTestMixin,
+    AcquiringTestMixin,
+} = require('@condo/domains/billing/utils/testSchema/testUtils')
+const { RUSSIAN_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { COLD_WATER_METER_RESOURCE_ID } = require('@condo/domains/meter/constants/constants')
+const { createTestMeterResourceOwner, updateTestMeterResourceOwner, MeterResource, MeterResourceOwner } = require('@condo/domains/meter/utils/testSchema')
+const { SERVICE_PROVIDER_TYPE } = require('@condo/domains/organization/constants/common')
+const { generateTin } = require('@condo/domains/organization/utils/testSchema')
+const { suggestServiceProviderByTestClient } = require('@condo/domains/resident/utils/testSchema')
+
+
+const addMeterResourceOwner = async (utils) => {
+    const resource = await MeterResource.getOne(utils.clients.admin, { id: COLD_WATER_METER_RESOURCE_ID })
+    return createTestMeterResourceOwner(utils.clients.admin, utils.organization, resource, {
+        address: utils.property.address,
+    })
+}
+const removeMeterResourceOwners = async (utils) => {
+    const owners = await MeterResourceOwner.getAll(utils.clients.admin, { organization: { id: utils.organization.id } })
+    for (const owner of owners) {
+        await updateTestMeterResourceOwner(utils.clients.admin, owner.id, { deletedAt: new Date().toISOString() })
+    }
+}
+
+describe('SuggestServiceProviderService', () => {
+
+    let utils
+
+    beforeAll(async () => {
+        utils = new TestUtils([ResidentTestMixin, MeterTestMixin])
+        await utils.init()
+        await utils.updateOrganization({ type: SERVICE_PROVIDER_TYPE })
+    })
+
+    describe('Permissions check', () => {
+
+        describe('Can not execute', () => {
+
+            test('anonymous: can not execute', async () => {
+                await expectToThrowAuthenticationErrorToResult(async () => {
+                    await suggestServiceProviderByTestClient(utils.clients.anonymous, {
+                        search: String(generateTin(RUSSIAN_COUNTRY)),
```

Comment:
no need to use RUSSIAN_COUNTRY, you can pass any string, as we only check for access rights